package.json

```
@@ -1,19 +1,15 @@
 {
   "name": "ngx-ui-switch",
-  "version": "16.1.0",
-  "description": "switch button for angular4",
+  "version": "20.1.3",
```

```suggestion
  "version": "16.1.0",
```

Revert version updates. Version updates are handled on master branch